agent-instrumentation/src/jvmMain/kotlin/com/epam/drill/agent/instrument/clients/OkHttp3ClientTransformerObject.kt

```diff
@@ -51,9 +55,11 @@ abstract class OkHttp3ClientTransformerObject : HeadersProcessor, AbstractTransf
             }
             """.trimIndent()
         )
-        ctClass.getDeclaredMethod("openResponseBody").insertCatching(
-            CtBehavior::insertBefore,
-            """
+        runCatching { ctClass.getDeclaredMethod("openResponseBody") }
```

Comment:
It's better to avoid catching exceptions in non-exceptional situations.
Here you can check for the existence of such a method using `getDeclaredMethods().any { it.name == methodName }`